src/app/global-vars.service.ts

```diff
@@ -838,7 +838,7 @@ export class GlobalVarsService {
 
   launchIdentityFlow(event: string): void {
     this.logEvent(`account : ${event} : launch`);
-    this.identityService.launch("/log-in", { referralCode: localStorage.getItem("referralCode") }).subscribe((res) => {
+    this.identityService.launch("/log-in?accessLevelRequest=4", { referralCode: localStorage.getItem("referralCode") }).subscribe((res) => {
```

Comment:
this wont work when referral is in local storage - query string will be ignored.

see the way diamond does it: https://github.com/diamond-app/frontend/blob/main/src/app/global-vars.service.ts#L990